doc/htmldoc/benchmark_results.rst

```diff
@@ -0,0 +1,106 @@
+.. _nest_benchmark_results:
+
+NEST performance benchmarks
+===========================
+
+
+NEST performance is continuously monitored and improved across various network sizes.
+Here we show benchmarking results for NEST version 3.8 on Jureca-DC.
+
+
+Strong scaling experiment of the Microcircuit model [1]_
+---------------------------------------------------------
+
+.. grid:: 1 1 1 1
+
+   .. grid-item::
+       :columns: 10
+       :class: sd-align-major-center
+
+       .. image:: /static/img/mc_benchmark.png
+
+.. grid:: 1 1 1 1
+
+   .. grid-item::
+       :columns: 10
+       :class: sd-align-minor-center
+
+
+       * The model has ~80 000 neurons and ~300 million synapses
+       * Increasing number of computing resources decrease simulation time
+       * The model runs faster than real time
```

Comment:
add reference [Z] at end of sentence so that the concept of "real time" can be looked up and there is a link for further information: Here is the reference:
Kurth, A. C. ;  Senk, J.; Terhorst, D.; Finnerty, J.; Diesmann, M. (2022) Sub-realtime simulation of a neuronal network of natural density. Neuromorphic computing and engineering  2(2), 021001 (2022) 
you can add doi link